transport/x/grpc/options.go

```diff
@@ -21,12 +21,24 @@
 package grpc
 
 import (
+	"math"
+
 	"github.com/opentracing/opentracing-go"
 	"go.uber.org/yarpc/api/backoff"
 	intbackoff "go.uber.org/yarpc/internal/backoff"
 	"google.golang.org/grpc"
 )
 
+const (
+	// defensive programming
+	// these are copied from grpc-go but we set them explicitly here
+	// in case these change in grpc-go so that yarpc stays consistent
+	defaultServerMaxRecvMsgSize = 1024 * 1024 * 4
+	defaultServerMaxSendMsgSize = math.MaxInt32
```

Comment:
> MaxSendMsgSize returns a ServerOption to set the max message size in bytes the server can send. If this is not set, gRPC uses the default 4MB.

I see all of these being 4MB in gRPC?

Reply:
I copied these on purpose, see comment on the variables